lib/earmark/inline.ex

```diff
@@ -44,17 +63,25 @@
     ]
   end
 
+  defmodule Link,   do: defstruct href: nil, text: nil, title: nil, ial: %{}
```

Comment:
I really like the idea, however I'd prefer a single file for each struct.

However that can easily be done later